inventory/byo/hosts.origin.example

```diff
@@ -255,6 +255,15 @@ openshift_master_identity_providers=[{'name': 'htpasswd_auth', 'login': 'true',
 # Can not be changed after deployment.
 #osm_cluster_network_cidr=10.1.0.0/16
 
+# ExternalIPNetworkCIDRs controls what values are acceptable for the
+# service external IP field. If empty, no externalIP may be set. It
+# may contain a list of CIDRs which are checked for access. If a CIDR
+# is prefixed with !, IPs in that CIDR will be rejected. Rejections
+# will be applied first, then the IP checked against one of the
+# allowed CIDRs. You should ensure this range does not overlap with
+# your nodes, pods, or service CIDRs for security reasons.
+#external_ip_network_cidrs=['0.0.0.0/0']
```

Comment:
Will need this mirrored in `inventory/byo/hosts.ose.example` and the variable should be `openshift_master_external_ip_network_cidrs`.